MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -58,30 +60,34 @@ public enum OfflineRoutingError: LocalizedError {
     }
 }
 
+/// :nodoc:
```

Comment:
Nit: `:nodoc:` is only needed for public symbols. jazzy doesn’t attempt to include private symbols in the docset anyways.